posthog/queries/sessions/session_recording.py

```diff
@@ -94,20 +103,32 @@ def filter_sessions_by_recordings(
     max_ts = max(it["end_time"] for it in sessions_results)
 
     session_recordings = query(team, min_ts, max_ts, filter)
+    viewed_session_recordings = set(
```

Comment:
I might be missing something but wouldn't it be better to delegate the heavy lifting of matching viewed session recordings to Postgres? As in get the list of sessions first and then do the select on the DB as opposed to getting the list of all viewed recordings every time?

Reply:
This wouldn't work for clickhouse as the two tables are stored in different dbs.